understanding/keyboard.html

```diff
@@ -55,248 +53,177 @@
-                  events that would collide with the modifier key lock to produce unexpected results.
-                  
+               <div class="note">
+                  <p class="note-title marker">訳注</p>
+                  <div><p>マウスキーとは、キーボードでマウスの操作を代替することのできるアクセシビリティ機能である。<a href="https://ja.wikipedia.org/wiki/%E3%83%9E%E3%82%A6%E3%82%B9%E3%82%AD%E3%83%BC">マウスキー - Wikipedia</a>も参照のこと。</p></div>
+               </div>
                   
+               <p>利用者が入力を行う機能の設計においては、オペレーティングシステムのキーボードアクセシビリティ機能が利用されている可能性を考慮することが期待される。例えば、修飾キーをロックする機能が有効になっているかもしれない。そのような環境でもコンテンツは機能し、修飾キーロックと衝突して予期しない結果を生じるようなイベントを送信しない (ようにすべきである)。
                </p>
                <div class="note">
-                  <p class="note-title marker">Note</p>
+                  <p class="note-title marker">訳注</p>
+                  <div><p>修飾キーとは、別のキーと組み合わせて使用し、そのキーの意味を一時的に変えるものである。例えば、Shift キーがこれに当たる。<a href="https://e-words.jp/w/%E4%BF%AE%E9%A3%BE%E3%82%AD%E3%83%BC.html">修飾キーとは - IT用語辞典</a>も参照のこと。</p></div>
+               </div>
+                  
+               <div class="note">
+                  <p class="note-title marker">注記</p>
                   <div>
                      
-                     <p>Platforms and user agents usually have conventions for how web content or
-                        applications are controlled with a keyboard interface. If content does not follow
-                        the platform/user agent conventions it may be difficult to use, as users will need
-                        to learn different interaction methods. As a <em>best practice</em>, content
-                        should follow the platform/user agent conventions. However, deviating from these
-                        conventions does <em>not</em> fail the normative requirement of this Success Criterion.
+                     <p>プラットフォーム及びユーザエージェントは通常、ウェブコンテンツ又はアプリケーションをキーボードインターフェイスでどう操作するかの慣習を持っている。コンテンツがプラットフォーム／ユーザエージェントの慣習に従わない場合、利用者は異なるインタラクション方法を学習しなければならず、使いにくくなる可能性がある。<em>ベストプラクティス</em>としては、コンテンツはプラットフォーム／ユーザエージェントの慣習に従うべきである。しかし、こうした慣習から逸脱しているからといって、この達成基準の規範的要件に失敗するとはならない。
                      </p>
                      
                      <p>
-                        For instance, buttons that have focus can generally be activated using both the
-                        <kbd>Enter</kbd> key and the <kbd>Space</kbd> bar. If a custom button control
-                        in a web application instead only reacts to <kbd>Enter</kbd>
-                        (or even a completely custom key or key combination), this still
-                        <strong>satisfies</strong> the requirements of this Success Criterion.
-                        
+                        例えば、フォーカスを持つボタンは、一般的に <kbd>Enter</kbd> キーと<kbd>スペースバー</kbd>のどちらを用いても実行することができる。ただしウェブアプリケーションのカスタムボタンコントロールが <kbd>Enter</kbd> キー (あるいは、完全にカスタムなキー又はキーの組み合わせ) にしか反応しない場合でも、この達成基準の要件を<strong>満たす</strong>ことになる。
```

Comment:
事例3でスペースキーとしていますので、そのようにしますか。
```suggestion
                        例えば、フォーカスを持つボタンは、一般的に <kbd>Enter</kbd> キーと <kbd>Space</kbd> キーのどちらを用いても実行することができる。ただしウェブアプリケーションのカスタムボタンコントロールが <kbd>Enter</kbd> キー (又は、完全にカスタムなキーもしくはキーの組み合わせ) にしか反応しない場合でも、この達成基準の要件を<strong>満たす</strong>ことになる。
```